rmp-serde/src/decode.rs

```diff
@@ -6,18 +6,9 @@ use std::result;
 use serde;
 
 use rmp::Marker;
-use rmp::decode::{
-    DecodeStringError,
-    FixedValueReadError,
-    MarkerReadError,
-    ReadError,
-    ValueReadError,
-    read_array_size,
-    read_numeric_data,
-    read_str_data,
-    read_marker,
-    read_full,
-};
+use rmp::decode::{DecodeStringError, FixedValueReadError, MarkerReadError, ReadError,
```

Comment:
Please revert style changes. There is no currently rustfmt config yet (it should be probably), but despite of code beautifulization these changes are out of the scope of this PR.